src/utilities/transform-mdx.tsx

```diff
@@ -1,19 +1,21 @@
-import * as Components from '@atom-learning/components/dist/index.modern'
+import * as Components from '@atom-learning/components'
 import hydrate from 'next-mdx-remote/hydrate'
 import renderToString from 'next-mdx-remote/render-to-string'
 import { MdxRemote } from 'next-mdx-remote/types'
 import * as React from 'react'
 
-import { CodeBlock, InlineCode, Link, List, Text } from '../components'
+import { CodeBlock, Divider, InlineCode, Link, List, Text } from '../components'
```

Comment:
couldn't we use the text and link from the components in line 1?

Very minor shouldn't hold the merge btw

Reply:
Yes absolutely, I'm going to make a separate PR  when the `@atom-learning/components` alpha is published and replace these and a few others